aws/variables.tf

```diff
@@ -1,3 +1,12 @@
+variable "account_name" {
+  type        = "string"
+  description = "AWS account name"
+}
+
+variable "environment" {
+  type    = "string"
+  default = "operations-shared"
+}
```

Comment:
Please allow for passing a variable to enable/disable creation fo S3 resources and ALB logging. Default false.